plugins/mygene/src/index.ts

```diff
@@ -0,0 +1,357 @@
+    },
+  },
+  { explicitlyTyped: true },
+)
+
+// translate thickStart/thickEnd to utr's
+// adapted from BigBedAdapter for ucsc thickStart/thickEnd
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function cdsStartEndProcessor(feature: any) {
+  // split the blocks into UTR, CDS, and exons
+  const { thickStart, thickEnd, strand, subfeatures: children } = feature
+
+  if (!thickStart && !thickEnd) {
+    return feature
+  }
+
+  const blocks = children
+    ? children.sort(
+        (a: { start: number }, b: { start: number }) => a.start - b.start,
+      )
+    : []
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const newChildren: Record<string, any> = []
+  blocks.forEach((block: { start: number; end: number }) => {
+    const { start, end } = block
+    if (thickStart >= end) {
+      // left-side UTR
+      const prime = strand > 0 ? 'five' : 'three'
+      newChildren.push({
+        type: `${prime}_prime_UTR`,
+        start,
+        end,
+      })
+    } else if (thickStart > start && thickStart < end && thickEnd >= end) {
+      // UTR | CDS
+      const prime = strand > 0 ? 'five' : 'three'
+      newChildren.push(
+        {
+          type: `${prime}_prime_UTR`,
+          start,
+          end: thickStart,
+        },
+        {
+          type: 'CDS',
+          start: thickStart,
+          end,
+        },
+      )
+    } else if (thickStart <= start && thickEnd >= end) {
+      // CDS
+      newChildren.push({
+        type: 'CDS',
+        start,
+        end,
+      })
+    } else if (thickStart > start && thickStart < end && thickEnd < end) {
+      // UTR | CDS | UTR
+      const leftPrime = strand > 0 ? 'five' : 'three'
+      const rightPrime = strand > 0 ? 'three' : 'five'
+      newChildren.push(
+        {
+          type: `${leftPrime}_prime_UTR`,
+          start,
+          end: thickStart,
+        },
+        {
+          type: `CDS`,
+          start: thickStart,
+          end: thickEnd,
+        },
+        {
+          type: `${rightPrime}_prime_UTR`,
+          start: thickEnd,
+          end,
+        },
+      )
+    } else if (thickStart <= start && thickEnd > start && thickEnd < end) {
+      // CDS | UTR
+      const prime = strand > 0 ? 'three' : 'five'
+      newChildren.push(
+        {
+          type: `CDS`,
+          start,
+          end: thickEnd,
+        },
+        {
+          type: `${prime}_prime_UTR`,
+          start: thickEnd,
+          end,
+        },
+      )
+    } else if (thickEnd <= start) {
+      // right-side UTR
+      const prime = strand > 0 ? 'three' : 'five'
+      newChildren.push({
+        type: `${prime}_prime_UTR`,
+        start,
+        end,
+      })
+    }
+  })
+  return { ...feature, subfeatures: newChildren, type: 'mRNA' }
+}
+
+// random notes for possible email to team: cdk11a/cdk11b return pretty bad data
```

Comment:
Email to our team or MyGene team?

Reply:
would be emailing mygene.info team. they dont return the right coordinates for those

Reply:
removed email for now

Reply:
er, will email team eventually...